src/plugins/intel_gpu/src/graph/impls/ocl/arg_max_min.cpp

```diff
@@ -48,7 +48,7 @@ struct arg_max_min_impl : typed_primitive_impl_ocl<arg_max_min> {
     kernel_arguments_data get_arguments(typed_primitive_inst<arg_max_min>& instance, int32_t) const override {
         kernel_arguments_data args = parent::get_arguments(instance, 0);
 
-        if (args.inputs.size() == 3) {
+        if (args.inputs.size() == 3 || args.outputs.size() == 2) {
```

Comment:
can this be changed to has_second_output?

Reply:
done